packages/frontend/src/components/Election/Sidebar.tsx

```diff
@@ -51,9 +51,21 @@ export default function Sidebar() {
                             {election.settings.voter_access != 'open' && <PermissionHandler permissions={permissions} requiredPermission={'canViewElectionRoll'}>
                                 <ListItem text='Voters' link={`/${id}/admin/voters`} />
                             </PermissionHandler>}
-                            <PermissionHandler permissions={permissions} requiredPermission={'canViewBallots'}>
-                                <ListItem text='Ballots' link={`/${id}/admin/ballots`} />
-                            </PermissionHandler>
+                            {/*
+                            The "Ballots" choice on the left-hand menu is available only if
+                            public_results is true and the election is open
+                            */}
+                            {election.settings.public_results = false}
+                            {
+                            (
```

Comment:
Hey sorry, I think I misspoke - your conditions were good last time except we don't need the ballot_updates condition. Something like:
```

                                !election.settings.public_results ||
                                election.state === 'closed'
                            ) &&
```

Reply:
sorry - ignore this, it's not correct